pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -96,6 +101,9 @@ public void init(SegmentGenerationJobSpec spec) {
           .generateTableConfigURI(pinotClusterSpec.getControllerURI(), _spec.getTableSpec().getTableName());
       _spec.getTableSpec().setTableConfigURI(tableConfigURI);
     }
+    int numThreads = JobUtils.getNumThreads(_spec.getSegmentCreationJobParallelism());
+    LOGGER.info("Creating an executor service with {} threads.", numThreads);
```

Comment:
Nit: It might be confusing for user on why num threads may not match the specified jobParallelism (if numCores is smaller). Perhaps modify the log indicate that.